Dockerfile

```diff
@@ -1,6 +1,8 @@
 ARG UBUNTU_VERSION=jammy
 
 FROM ubuntu:$UBUNTU_VERSION AS build-environment
+ARG PLUGIN_INPUT=All
+ENV PLUGIN_INPUT=${PLUGIN_INPUT}
```

Comment:
You do not need an ARG and the ENV. Should be enough just to have the ARG. See J2735_VERSION